index.bs

```diff
@@ -2972,14 +2979,16 @@ the same operation or constructor.
     identifier <code>f</code> and argument count 4 is:
 
     <pre class="set">
-      { &lt;f1, (DOMString), (required)&gt;,
-        &lt;f2, (Node, DOMString), (required, required)&gt;,
-        &lt;f2, (Node, DOMString, double), (required, required, variadic)&gt;,
-        &lt;f2, (Node, DOMString, double, double), (required, required, variadic, variadic)&gt;,
-        &lt;f3, (), ()&gt;,
-        &lt;f4, (Event, DOMString), (required, required)&gt;,
-        &lt;f4, (Event, DOMString, DOMString), (required, required, optional)&gt;,
-        &lt;f4, (Event, DOMString, DOMString, double), (required, required, optional, variadic)&gt; }
+      « 
+        (f1, « DOMString »,                           « required »),
```

Comment:
I wonder if we should change the optionality values to have quotes around them, since technically that's what we do above. Or maybe it's just noise.